docs/usage-guide/topics/ch04-config.md

```diff
@@ -1,5 +1,7 @@
 # Configuring the Connection
 
+Users start by building a config. In the general case, there will be one config per server. This involves loading the certificate, configuring session resumption, etc. Users should configure an `s2n_config` before associating any `s2n_connection` objects with it.
```

Comment:
> Users should configure an `s2n_config` before associating any `s2n_connection` objects with it.

This is also already stated:

> A config should not be altered once it is associated with a connection as this will produce undefined behavior.

Reply:
These two sentences are not the same. A better way is to rephrase the second quote to make it clearer.